compiler/rustc_lint/src/lints.rs

```diff
@@ -1210,9 +1210,10 @@ impl<'a> DecorateLint<'a, ()> for DropGlue<'_> {
 #[diag(lint_range_endpoint_out_of_range)]
 pub struct RangeEndpointOutOfRange<'a> {
     pub ty: &'a str,
-    #[suggestion(code = "{start}..={literal}{suffix}", applicability = "machine-applicable")]
-    pub suggestion: Span,
-    pub start: String,
+    #[suggestion(code = "=", applicability = "machine-applicable")]
+    pub eq_suggestion: Span,
+    #[suggestion(code = "{literal}{suffix}", applicability = "machine-applicable")]
+    pub lit_suggestion: Span,
```

Comment:
This makes two suggestions, which indeed looks weird. Instead you should use multipart suggestions (you can search for `#[multipart_suggestion(` to find examples on how to do this).

Reply:
Got it! Thanks!